tests/fsharp/typecheck/sigs/neg_anon_1.bsl

```diff
@@ -1,5 +1,5 @@
 
-neg_anon_1.fs(5,50,5,61): typecheck error FS0001: This anonymous record does not have enough fields. Add the missing fields [b].
+neg_anon_1.fs(5,50,5,61): typecheck error FS0001: This anonymous record is missing field b.
```

Comment:
Sorry for nitpicking, I am just thinking that in this current shape it can cause some funny confusion if the fields are given some meaningful names. As in:
- "This anonymous record is missing field maybe"
- "This anonymous record is missing field name"
- "This anonymous record is missing field mice"

Probably some quote/escape would be useful - as you wish though.